old_table_1:
  x_min: -0.4
  x_max: 0.4
  y_min: -0.4
  y_max: 0.4
  z_min: 0.695
  z_max: 0.72

table_1:
  x_min: -0.35
  x_max: 0.35
  y_min: -0.715
  y_max: 0.715
  z_min: 0.0
  z_max: 0.72

table_2:
  x_min: -0.715
  x_max: 0.715
  y_min: -0.35
  y_max: 0.35
  z_min: 0.0
  z_max: 0.72

table_3:
  x_min: -0.715
  x_max: 0.715
  y_min: -0.35
  y_max: 0.35
  z_min: 0.0
  z_max: 0.72

power_drill_1:
  x_min: -0.09
  x_max: 0.09
  y_min: -0.11
  y_max: 0.11
  z_min: -0.04
  z_max: 0.04

power_drill_with_grip_1:
  x_min: -0.09
  x_max: 0.09
  y_min: -0.11
  y_max: 0.11
  z_min: -0.04
  z_max: 0.04

screwdriver_1:
  x_min: -0.12
  x_max: 0.12
  y_min: -0.018
  y_max: 0.018
  z_min: -0.018
  z_max: 0.018

relay_1:
  x_min: -0.028
  x_max: 0.028
  y_min: -0.0225
  y_max: 0.0225
  z_min: -0.05
  z_max: 0.05

multimeter_1:
  x_min: -0.0885
  x_max: 0.0885
  y_min: -0.043
  y_max: 0.043
  z_min: -0.021
  z_max: 0.021

klt_1:
  x_min: -0.15
  x_max: 0.15
  y_min: -0.1
  y_max: 0.1
  z_min: -0.074
  z_max: 0.074

klt_2:
  x_min: -0.15
  x_max: 0.15
  y_min: -0.1
  y_max: 0.1
  z_min: -0.074
  z_max: 0.074